server/service/integrationtest/android/android_test.go

```diff
@@ -0,0 +1,48 @@
+package android
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/datastore/mysql"
+	"github.com/fleetdm/fleet/v4/server/mdm/android"
+	"github.com/fleetdm/fleet/v4/server/service/integrationtest"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAndroid(t *testing.T) {
+	s := integrationtest.SetUpSuite(t, "integrationtest.Android")
+
+	cases := []struct {
+		name string
+		fn   func(t *testing.T, s *integrationtest.Suite)
+	}{
+		{"HappyPath", testHappyPath},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			defer mysql.TruncateTables(t, s.DS)
```

Comment:
My guess is that at some point we may need more than truncating the DB tables as cleanup between tests (e.g. there's some state that persists with all the stuff created from the SetupSuite call), but we can figure those out as we go (would be important early on to run tests in isolation or in various order to ensure we catch any cross-test-dependency - one suggestion that could help identify that would be to shuffle the test `cases` slice, and print it's order in case it fails so we can reproduce.